crypto/bio/bio.c

```diff
@@ -803,3 +808,23 @@ void BIO_set_callback_arg(BIO *bio, char *arg) {
 char *BIO_get_callback_arg(const BIO *bio) {
   return bio->cb_arg;
 }
+
+int BIO_get_ex_new_index(long argl, void *argp,
```

Comment:
Do we have any existing tests that might be extended to cover this function?

Reply:
We'll probably have to write a new one, this functionality is different from other BIO tests. It's extending upon an [existing family of functions/structures](https://github.com/search?q=repo%3Aaws%2Faws-lc%20CRYPTO_get_ex_new_index&type=code) which define external data though.

Reply:
Test added :)